store/sqlstore/reaction_store.go

```diff
@@ -110,22 +123,36 @@ func (s *SqlReactionStore) BulkGetForPosts(postIds []string) ([]*model.Reaction,
 
 func (s *SqlReactionStore) DeleteAllWithEmojiName(emojiName string) error {
 	var reactions []*model.Reaction
+	now := model.GetMillis()
+
+	params := map[string]interface{}{
+		"EmojiName": emojiName,
+		"UpdateAt":  now,
+		"DeleteAt":  now,
+	}
 
 	if _, err := s.GetReplica().Select(&reactions,
 		`SELECT
-				*
+				UserId,
+				PostId,
+				EmojiName,
+				CreateAt,
+				COALESCE(UpdateAt, CreateAt) As UpdateAt,
+				COALESCE(DeleteAt, 0) As DeleteAt
```

Comment:
We can still use `*` here, can't we? Because in the for loop later, we only use the `PostId`. UpdateAt and DeleteAt are completely ignored.

Reply:
Gorp still has to convert the NULL to a Int64 to fill the struct and that is where it fails on NULL.  I made the unit test force NULLs just in case someone sets it back to `*` at some point.